src/main/java/picard/sam/MergeBamAlignment.java

```diff
@@ -280,6 +301,16 @@
             "the result will be an invalid record. To force no change use the DO_NOT_CHANGE_INVALID strategy.", optional = true)
     public AbstractAlignmentMerger.UnmappingReadStrategy UNMAPPED_READ_STRATEGY = AbstractAlignmentMerger.UnmappingReadStrategy.DO_NOT_CHANGE;
 
+    @Argument(doc = "Structure of read 1.  If this and READ_STRUCTURE both are unspecified, will be assumed that all bases in read 1 are template. Cannot be used with READ_STRUCTURE.", optional = true, mutex = {"READ_STRUCTURE"})
+    public ReadStructure READ_1_STRUCTURE;
```

Comment:
Why do we need two read structures? the read structure is something that breaks up the entire stream of cycles into various things: UMIs,  Sample BCs, Template reads, skips...it is not clear (to me!) how a single readStructure should be broken up into to.